client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -238,9 +243,17 @@ const EmailPlan = ( props ) => {
 	}
 
 	function renderAddNewMailboxesOrRenewNavItem() {
-		const { domain, hasSubscription, purchase, translate } = props;
-
-		if ( hasTitanMailWithUs( domain ) && ! hasSubscription ) {
+		const {
+			canAddGoogleWorkspaceMailboxes,
+			canAddProfessionalEmailMailboxes,
+			domain,
+			hasSubscription,
+			purchase,
+			translate,
+		} = props;
+
+		const canAddMailboxes = canAddGoogleWorkspaceMailboxes || canAddProfessionalEmailMailboxes;
```

Comment:
Since we actually never use `canAddGoogleWorkspaceMailboxes` nor `canAddProfessionalEmailMailboxes` in isolation it seems we could simplify the code, and pass `canAddMailboxes` in the `connect()` block.

Reply:
I had the same feeling but I thought it would be good for future. But fair enough, I'm merging it into one prop.